examples/basics/plotting/grid_x_y_viewbox.py

```diff
@@ -0,0 +1,47 @@
+
+
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
++---+-------------+
+|   |             |
+| y |   viewbox   |
+|   |             |
++---+-------------+
+| sp|     x       |
++---+-------------+
```

Comment:
Looks good. But for some strange reason on my system it doesn't always recalculate / reposition if I resize the window multiple times in succession. Is it always reliable for you? Could just be an event-system bug, so probably not a show-stopper.